Camera ready

raphlinus opened the pull request:
This is a working draft of the "camera ready" version. The final document numbers will be added on the ACM side.

I think this is submittable as is. One thing I see that I would like to address is the bitmap "g" in Fig 8. Also, I'm not completely satisfied with Fig 11, but I might call it "good enough."

armansito committed:
ACM formatting fixes

raphlinus committed:
Minor formatting tweaks

Apply acmauthoryear cite style as requested. Small changes to the figures, making lines bolder and adding axis labels.

Also add a mention to the Vello rendering engine, which was missing due to anonymity.

armansito commented on paper.tex:
```diff
@@ -35,9 +36,9 @@
-%\conferenceinfo{Foo}{Month Day-Day, Year, Location}
+\copyrightyear{2024}
+\setcopyright{acmcopyright}
+%\conferenceinfo{High Performance Graphics}{July 26-28 2024, Denver}
```
Should this be uncommented?

raphlinus replied:
That macro is not defined in the template, so uncommenting it causes an error. I believe the metadata is set correctly now, see eaac725.

raphlinus committed:
Apply ACM bibliographic metadata

This applies the DOI and the rights statement as appropriate for publication in PACMCGIT.

raphlinus committed:
Replace bitmap rendering of g in figure

Use a vector rendering of the original Nimbus 'g' in the figure, rather than a bitmap. Visually it is very similar, just crisper.

raphlinus commented:
Marking as ready to review, as this is the exact version sent to ACM. There's a short window yet to make revisions, but I believe we should consider this the final version.

armansito reviewed (commented):
Latest notes from Stephen:

Please add your e-mail addresses using the \email{} command.

The headers and footers are missing, and there are page numbers.

Include \citestyle{acmauthoryear} in the preamble.

Algorithm 2 exceeds the page margins (at the bottom).

raphlinus commented:
Yep, working on those notes. Some are straightforward, we need `\citestyle{acmauthoryear}` rather than `\setcitestyle{acmauthoryear}`, the latter doesn't seem to do anything. I'm figuring out headers and page numbers now.

raphlinus committed:
More formatting fixes

Use acmauthoryear cite style. Add email addresses. Squash linespace of algorithm 2 to fit page margins.

With this cite style, the `\citenum` command is no longer valid, so those are all changed to `\cite`. There are a number of instances that use `\citet` with the author specified in the text, and I believe those are compliant, but I'm not sure that's best. It's possible those should be edited to `\cite` so the author is inside the brackets.

raphlinus committed:
Use \cite consistently

Just use `\cite` for all citations, as there's no longer a need to distinguish `\citenum` from `\citet`.

Minor text edits have been made to make this consistent, and the authors of a couple corporately authored artifacts have been changed to mangle the names less.
